wordless/theme_builder/vanilla_theme/webpack.config.js

```diff
@@ -7,6 +7,8 @@ const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ImageminWebpack = require('image-minimizer-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const imageFolder = 'images'
```

Comment:
I'd prefere to name it `imageFolderName` in order to distinguish it from the various paths already set up.

Alternatively having a `imagesDstPath` and a `imagesSrcPath` set up to the complete path could be another valid approach and would follow the previously adopted practice. But this could be just overwhelming.